src/transformers/tokenization_utils_fast.py

```diff
@@ -60,6 +62,12 @@
             from 🤗 tokenizers <../fast_tokenizers>` for more information.
 """
 
+MODEL_TO_TRAINER_MAPPING = {
+    "BPE": BpeTrainer,
+    "Unigram": UnigramTrainer,
+    "WordPiece": WordPieceTrainer,
+}
+
```

Comment:
```suggestion
MODEL_TO_TRAINER_MAPPING = {
    "BPE": BpeTrainer,
    "Unigram": UnigramTrainer,
    "WordLevel": WordLevelTrainer,
    "WordPiece": WordPieceTrainer,
}

```

Reply:
I have the impression that none of the fast tokenizers we test use a `WordLevel` tokenizer model, but if that were the case I think we would need this extra line.

Maybe we should add a test that uses such a tokenizer? For example, in the fast tokenizer tests?